### ISSUE TITLE
support rendering multiple forms

### ISSUE THREAD

**Issue by mattyg** [pragmatic-engineering/svelte-form-builder-community#11]
Thanks for this great library. I just wanted to request support for rendering multiple forms on the same page. Currently if you try this you get a race condition or something that causes the browser to freeze / crash.

**Comment by Jojoshua** [comment]
Multiple forms was intended (single instance with multiple tabs and multiple separate instances). Can you show the minimal repro code?

**Comment by mattyg** [comment]
wow thanks for the quick response! Here's a reproduction: https://github.com/mattyg/svelte-form-builder-multiple-components-bug-reproduction

The page `/working` only loads one FormBuilder and runs properly.

The home page loads 2 FormBuilders and slows the browser to a halt.

**Comment by Jojoshua** [comment]
I will have to test some more later but looking at the code I see both instances are set to **view: "build"**. I think the expectation was that there would only be a single view needed for the user to build each form, then the config could be saved and passed into the rendered pages where multiple **view: "render"** instances would be used. Build mode is needed only once (developing a config) while render mode (consuming the configs for production) is switching hats. Does that make sense and can you try it?

**Comment by mattyg** [comment]
Yeah the issue occurs in render mode too -- I just modified the repro to demonstrate. 

I think the underlying issue related to keeping initialization-related data in a svelte writable store, so multiple rendered components are reading / writing to the same store and interfering with each others' initialization progress. That data should probably just live in the component itself, so each instance has its own copy.

**Comment by Jojoshua** [comment]
Thanks for checking. I will take a look into that. Feel free to send any suggested PRs in the meantime.
